ipaplatform/redhat/services.py

```diff
@@ -73,6 +73,7 @@
 redhat_system_units['ods-signerd'] = 'ods-signerd.service'
 redhat_system_units['ods_signerd'] = redhat_system_units['ods-signerd']
 redhat_system_units['gssproxy'] = 'gssproxy.service'
+redhat_system_units['sssd'] = 'sssd.service'
```

Comment:
This assignment is not required. You only need to override system units when the service name does not match the service file. Please remove.